### PR TITLE
Mention that bbcode_text += string is a bad idea

### LINKED ISSUE #16439: RTL.bbcode_text += operator doesn't use append_bbcode
should use `append_bbcode`, currently it uses `set_bbcode` internally if doing `+=` operator on `bbcode_text` property.

 (set_bbcode causes massive lag when adding a ton of text over time to the `"bbcode_text"` property)

@willnationsdev is current rtl guru, plz take a look, ty sir.
https://github.com/godotengine/godot/blob/e24f2c6489afdccc6268008d3dcc1ff4ad8ea71c/scene/gui/rich_text_label.cpp#L2044

just to add: possible internal issue with gdscript's `+=` operator, since it's a string appendation.  
might need a special case for `bbcode_text` appending, and have it use `append_bbcode` instead

### PR DESCRIPTION
**Pull request by KoBeWi**
Resolves #16439

### CODE REVIEW

**Review by iatenine — APPROVED** [review]
I was unaware of this and Godot's documentation on menus and dialogue functions is sorely lacking compared to other areas. This is useful info

File: doc/classes/RichTextLabel.xml [hunk 1]
```diff
@@ -278,6 +278,7 @@
 		</member>
 		<member name="bbcode_text" type="String" setter="set_bbcode" getter="get_bbcode" default="&quot;&quot;">
 			The label's text in BBCode format. Is not representative of manual modifications to the internal tag stack. Erases changes made by other methods when edited.
+			[b]Note:[/b] it is unadvised to use [code]+=[/code] operator with [code]bbcode_text[/code] (e.g. [code]bbcode_text += "some string"[/code]) as it replaces whole text and can cause slowdowns. Use [method append_bbcode] for adding text instead.
```
Comment (akien-mga) [inline doc/classes/RichTextLabel.xml]: replaces **the** whole text

Comment (akien-mga) [reply doc/classes/RichTextLabel.xml]: I'd also start with capital "I" after "Note:", as we do in other places.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**